packages/client/src/arpa_reporter/views/NewUpload.vue

```diff
@@ -1,111 +1,164 @@
 <template>
-  <div class="upload">
-    <h1>Submit Workbook</h1>
+  <div>
 
-    <div>
-      <div v-if="error" class="mt-3 alert alert-danger" role="alert">
-        {{ error }}
-      </div>
-
-      <form
-        method="post"
-        encType="multipart/form-data"
-        ref="form"
-        @submit.prevent="onSubmit"
-      >
-        <div class="form-group">
-          <input
-            class="form-control-file"
-            type="file"
-            id="spreadsheet"
-            name="spreadsheet"
-            @change="changeFiles"
-            ref="files"
-          />
-        </div>
-        <div class="form-group">
-          <button
-            class="btn btn-primary"
-            type="submit"
-            :disabled="uploadDisabled"
-            @click.prevent="onSubmit"
-          >
-            {{ uploadButtonLabel }}
-          </button>
-          <a class="ml-3" href="#" @click="cancelUpload">Cancel</a>
-        </div>
-      </form>
-    </div>
+    <FormulateForm
+      name="new-upload"
+      :form-errors="formErrors"
+      v-model="values"
+      :schema="schema"
+      @submit="onSubmit"
+    >
+    <FormulateErrors
+      class="alert alert-danger"
+    />
+    </FormulateForm>
   </div>
 </template>
 
 <script>
+import Vue from 'vue';
+import VueFormulate from '@braid/vue-formulate';
 import { postForm } from '../store';
 
+Vue.use(VueFormulate, {
+  classes: {
+    prefix: 'formulate-',
+    outer: 'form-group',
+    input: (context, classes) => {
+      // set this to 'form-control' unless it's a file input
+      const inputClass = context.type === 'file' ? '' : 'form-control';
+      return [classes, inputClass].join(' ');
```

Comment:
According to the documentation, if we return an array here, it will automatically join the new input class with existing classes.

```suggestion
      return [inputClass];
```